File: Project/Backend/daariiou_photography_backend/daariiou_photography_backend/DTO/PictureDTO.cs
using daariiou_photography_backend.Model;
using System;
using System.Collections.Generic;

#nullable disable

namespace daariiou_photography_backend.DTO
{
    public class PictureDTO
    {
        public int Pid { get; set; }
        public byte[] Src { get; set; }
        public string IsPublic { get; set; }
        public DateTime Date { get; set; }
        public int? Uid { get; set; }
        public byte[] Thumb { get; set; }
        public int KoSid { get; set; }
        public User UidNavigation { get; set; }
        public Model.KindOfShooting KoS { get; set; }
    }
}

File: Project/Frontend/daariiou-photography-frontend/src/api/lib/models/picture-dto.ts
/* tslint:disable */
/* eslint-disable */
import { KindOfShooting } from './kind-of-shooting';
import { User } from './user';
export interface PictureDto {
  date?: string;
  isPublic?: null | string;
  koS?: null | KindOfShooting;
  koSid?: number;
  pid?: number;
  src?: null | any;
  thumb?: null | string;
  uid?: null | number;
  uidNavigation?: null | User;
}
